roles/download/defaults/main.yml

```diff
@@ -110,7 +110,7 @@ flannel_cni_version: "v1.0.1"
 cni_version: "v1.0.1"
 weave_version: 2.8.1
 pod_infra_version: "3.3"
-cilium_version: "v1.11.1"
+cilium_version: "v1.11.3"
```

Comment:
Just question: There is a lot of cilium code which support older versions of cilium (1.8-)
Do we still need to keep those code?

It is fine to keep those code in this pull request for keeping the pull request small.
I just wanted to make sure we will remove the code or not soon.